src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An Person attribute that can be printed as a String
+ */
+public interface Printable {
```

Comment:
Good job! you can also try use getPrintableString method somewhere (you could think of  a good place to put it), so it's more meaningful.